atomic_reactor/core.py

```diff
@@ -301,6 +334,7 @@ def build_image_from_git(self, url, image, git_path=None, git_commit=None,
         logger.info("build finished")
         return response
 
+    @retry
```

Comment:
This method uses two Docker API calls, so if the second one (start the container) fails the first one (create the container) will be executed again. I think we probably want to retry at the API call level, i.e. a retry class wrapping docker.APIClient, rather than at the level of methods of DockerTasker.